realscale.inx

```diff
@@ -23,9 +95,11 @@
 
 • Select the line first, then the drawing (which must be a group or a single object).
 
-• Unit: indicate the unit you would like to use (e.g. mm, cm, ...), but remember that your drawing can become huge if you choose m or even km. Consider replacing m with cm for convenience.
+• Length: indicate how long the line you drew is in the real world (e.g. how long the wall is).
+
+• Unit: indicate the unit the measurement is in, but remember that your drawing can become huge if you choose km or even m. Fix this with Drawing Scale.
 
-• Length: indicate how long the line you drew is in the real world (e.g. how long the wall is).</_param>
+• Drawing Scale: This will scale the drawing to allow you to fit it on the page. You can show this scale with a ruler.</_param>
```

Comment:
There isn't any item named 'Drawing Scale'. I think it's possible to add a subheading into the 'form', as is done with 'Usage' - maybe that would be an option, if you don't want to explain all the options separately?